frontend/src/pages/landing/landing.module.scss

```diff
@@ -0,0 +1,36 @@
+.container {
+  position: relative;
+  width: 100%;
+  height: 100vh;
+  overflow: hidden;
+}
```

Comment:
I guess this styling could be defined to div in `App` component (except position and overflow) as we may apply full screen sizing for all pages

Reply:
Resolved - [120c4b7](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/120c4b7f43399e997ec85e48fb3635e2eb2b154d) + [75f0ef2](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/26/commits/75f0ef2575282818a2f0fe014adcb41a2c026f12)